Extension/src/LanguageServer/extension.ts

```diff
@@ -1174,32 +1178,48 @@ async function handleCrashFileRead(crashDirectory: string, crashFile: string, er
                 funcStr = funcStr.replace(/, std::allocator<std::string>/g, "");
             }
         }
-        data += funcStr + offsetStr;
+        crashCallStack += funcStr + offsetStr;
         const offsetPos2: number = offsetPos + offsetStr.length;
         if (isMac) {
-            data += line.substring(offsetPos2);
+            crashCallStack += line.substring(offsetPos2);
             const startAddressPos: number = line.indexOf("0x");
             if (startAddressPos === -1 || startAddressPos >= startPos) {
                 // unexpected
-                data += "<Missing 0x>";
+                crashCallStack += "<Missing 0x>";
                 continue;
             }
             addressData += `${line.substring(startAddressPos, startPos)}`;
         } else {
             const endPos: number = line.indexOf(">", offsetPos2);
             if (endPos === -1) {
-                data += "<Missing > >";
+                crashCallStack += "<Missing > >";
                 continue; // unexpected
             }
-            data += line.substring(offsetPos2, endPos);
+            crashCallStack += line.substring(offsetPos2, endPos);
+        }
+    }
+
+    if (crashCallStack !== prevCppCrashCallStackData) {
+        prevCppCrashCallStackData = crashCallStack;
+
+        const settings: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("C_Cpp", null);
+        if (lines.length >= 6 && util.getNumericLoggingLevel(settings.get<string>("loggingLevel")) >= 1) {
+            const out: vscode.OutputChannel = getCrashCallStacksChannel();
+            out.appendLine("");
+            out.append(localize({ key: "crash.callstack.info", comment: [
+                "{0} is the process name: either cpptools or cpptools-srv. {1} is the crash date and time, e.g. 4/24/2024, 4:44:52 PM. {2} is a signal name, e.g. SIGSEGV. The call stack lines appear on a new line after the ':'" ]},
+            "The {0} process crashed on {1} from signal {2} with call stack:",
```

Comment:
@bobbrow @Colengms Is the "sentence" format okay or would an undescriptive version be better, with the description being in the one time description at that top?

```txt
cpptools
4/24/2024, 4:44:52 PM
SIGSEGV
call_stacks...
```

Reply:
This suggestion seems to make it easier for the localization team.  Either way is fine with me though.

As a general recommendation, I avoid putting punctuation like the trailing `:` in the strings (when it makes sense) so that they don't have the ability to change the character.

Reply:
Sure, I'll go with the one-time description, but I'm not understanding your suggestion in regards to ":" -- they always have the potential to change that punctuation and that seems fine to me, since other languages have different punctuation guidelines.

Reply:
Oh, I think I'm getting rid of the trailing ":" due to re-wording anyways.